apps/docs/content/guides/platform/read-replicas.mdx

```diff
@@ -168,3 +199,7 @@ In this replication method, the Primary continuously buffers WAL changes to a lo
 We bring these two methods together to achieve quick, stable, and reliable replication. Each method addresses the limitations of the other. Streaming replication minimizes replication lag, while file-based log shipping provides a fallback. For file-based log shipping, we use our existing Point In Time Recovery (PITR) infrastructure. We regularly archive files from the Primary using [WAL-G](https://github.com/wal-g/wal-g), an open source archival and restoration tool, and ship the WAL files to S3.
 
 We combine it with streaming replication to reduce replication lag. Once WAL-G files have been synced from S3, Read Replicas connect to the Primary and stream the WAL directly.
+
+### Monitoring replication lag
```

Comment:
Can we mention the metric they can monitor for this as well (in their own metrics infra), and also recommend the alarms to set up?